apps/fluent-tester/src/TestComponents/Avatar/BasicAvatar.android.tsx

```diff
@@ -3,109 +3,145 @@ import { Avatar } from '@fluentui-react-native/avatar';
 import { View } from 'react-native';
 import { steveBallmerPhotoUrl } from './../PersonaCoin/styles';
 import { mobileStyles } from '../Common/styles';
+import { Switch } from '@fluentui-react-native/switch';
+import { InteractionEvent } from '@fluentui-react-native/interactive-hooks';
 
 export const StandardUsage: FunctionComponent = () => {
-  return (
-    <View style={{ paddingBottom: 40 }}>
-      <View style={mobileStyles.testVariants}>
-        {/* Variation - Standard */}
+  const [activityRing, setActivityRing] = React.useState(true);
 
-        {/* With badge. */}
-        <Avatar badge={{ status: 'available' }} avatarColor={'brand'} size={40} />
+  const toggleActivityState = (_e: InteractionEvent, isActive: boolean) => {
+    setActivityRing(isActive);
+  };
 
-        {/* Variation - No Ring and badge. */}
-        <Avatar size={56} avatarColor={'brand'} />
+  return (
+    <View style={{ paddingBottom: 10, paddingTop: 5 }}>
```

Comment:
nit: Please avoid object literals as props

Reply:
Updated to refer to a const in style file.